core/core-funcs.el

```diff
@@ -277,4 +277,77 @@ buffer."
           . (,feature . ,repl-func))
         spacemacs-repl-list))
 
+(defun spacemacs/orgify-orgbuf ()
```

Comment:
It might be a good idea to split the function and reuse its parts in  [spacemacs/org-githubify-orgbuf](https://github.com/syl20bnr/spacemacs/pull/5342/files#diff-c1f53f6886fdfa85b5282798acbc2f1eR11)